.pipelines/pipeline.yaml

```diff
@@ -322,24 +322,39 @@ stages:
       displayName: Cilium
       pipelineBuildImage: "$(BUILD_IMAGE)"
       testDropgz: ""
+      clusterType: swift-no-kube-proxy-up
       clusterName: "ciliume2e"
+      vmSize: Standard_B2ms
+      k8sVersion: ""
```

Comment:
If we are sending anything in the parameters, we can leave them out ?

Reply:
If we don't pass anything the container template fails. The ${{ parameters.* }} calls don't result in NULL as expected if the corresponding parameter doesn't pass anything into the make "cluster-type" command. 

A work around for this to prevent clutter would be to have various make "cluster-type" command based on os, arch, ect. That would prevent us having to pass in k8sVersion ect. when t is not being used.